infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -56,16 +60,50 @@ def run_shell_command(command, env_vars):
 ###################
 
 
-def make_backup_fpath():
-    return BACKUP_DIR / datetime.now().strftime(FILENAME_DATETIME_FORMAT)
+def make_object_name():
```

Comment:
I think this function name is a little too vague. It looks like this is returning the current timestamp as a string. Was there not already a function that did this? At any rate, we could name this function: `get_formatted_current_timestamp()`.

Reply:
Where the return type is `str`, obviously 🤣